ingenialink/ethercat/network.py

```diff
@@ -144,6 +134,30 @@ def __init__(
         self.__is_master_running = False
         self.__last_init_nodes: list[int] = []
 
+    @staticmethod
+    def __get_foe_error_message(error_code: int) -> str:
+        """Error message associated with an error code.
+
+        Args:
+            error_code: FoE error code.
+
+        Returns:
+            Error message.
+        """
+        # Error codes taken from SOEM source code.
+        # https://github.com/OpenEtherCATsociety/SOEM/blob/v1.4.0/soem/ethercatfoe.c#L199
+        if error_code == -3:
+            return "Unexpected mailbox received"
+        if error_code == -5:
+            return "FoE error"
+        if error_code == -6:
+            return "Buffer too small"
+        if error_code == -7:
+            return "Packet number error"
+        if error_code == -10:
+            return "File not found"
+        return f" Error code: {error_code}."
```

Comment:
Why do you not reuse the dictionary? For me, using a dictionary is clearer than using a lot of if statements.